presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -635,6 +661,11 @@ as part of a SQL query by including them in your SELECT statement.
      ----------------------------------+------------
                   2                    | 3
 
+Presto C++ Support
+^^^^^^^^^^^^^^^^^^
+
+All above extra hidden metadata columns are supported in Presto C++.
```

Comment:
```suggestion
All above metadata columns are supported in Presto C++.
```

> above extra hidden

Three adjectives in a row feels like a lot. I think we can reduce

Reply:
Here existing `extra hidden metadata columns` is taken from the main heading. But if we think we should just rephrase it saying - `All above metadata columns are supported in Presto C++` that's fine with me?

Reply:
It feel overly description to have three adjectives in the sentence - I think this gets the point across